blockpage.js

```diff
@@ -56,7 +56,7 @@ function fetchProtectionRules(url,status){
         addRules: domains.map((domain, index) => ({
           id: index + 1,
           priority: 1,
-          action: { type: "redirect", redirect: { url: "https://google.com/" } },
+          action: { type: "redirect", redirect: { url: chrome.runtime.getURL("warning_page.html") } },
```

Comment:
Can we change this so that it memorises the page they were on and redirects them to that page when they pause NMD?

Reply:
Sounds good.